projects/media-viewer/src/lib/media-viewer/annotation/annotation-set.model.ts

```diff
@@ -0,0 +1,127 @@
+export interface IComment {
+  id: string;
+  annotationId: string;
+  createdBy: string;
+  createdByDetails: IdamDetails;
+  createdDate: any;
+  lastModifiedBy: string;
+  lastModifiedByDetails: IdamDetails;
+  lastModifiedDate: any;
+  content: string;
+}
+
+export class Comment implements IComment {
```

Comment:
should we put the classes/interfaces in separate locations, so that we can export the interfaces separately from the concrete implementations